CYBAMAN/CYBAMAN-IOC-01App/Db/cybaman.db

```diff
@@ -26,6 +26,7 @@ record(bo, "$(P)INITIALIZE")
 	field(OUT, "@devcybaman.proto initialize $(PORT)")
 	field(FLNK, "$(P)_SET_INITIALIZED_TRUE")
 	info(INTEREST, "MEDIUM")
+	field(alarm, "CYBAMAN")
```

Comment:
This is not reading from the device so this isn't a good record to put an alarm on.

I think a better PV would be one of the axes (look in `cybaman_axes.db`)